file_read_backwards/file_read_backwards.py

```diff
@@ -40,34 +38,32 @@ def __init__(self, path, encoding="utf-8", chunk_size=io.DEFAULT_BUFFER_SIZE):
         self.path = path
         self.encoding = encoding.lower()
         self.chunk_size = chunk_size
-        self.__fp_refs = []
+        self.iterator = FileReadBackwardsIterator(io.open(self.path, mode="rb"), self.encoding, self.chunk_size)
```

Comment:
I think it should be
`self.iterator = None`

and then in __iter__(self):
```python
    def __iter__(self):
        """Return its iterator."""
        if self.iterator is None:
            self.iterator = FileReadBackwardsIterator(io.open(self.path, mode="rb"), self.encoding, self.chunk_size)
        return self.iterator
```
This way, we hit the FS only when frb starts being used.

Reply:
Thanks. I'l check the behavior of Python's `file`.